README.adoc

```diff
@@ -0,0 +1,142 @@
+= Hazelcast Viridian Maven Plugin
+:description: The Hazelcast Viridian Maven Plugin is a Java development tool for testing and deploying cluster-side modules on Serverless clusters.
+:page-plugin-version: 0.4-SNAPSHOT
```

Comment:
The latest released version to Maven Central is `0.0.1`

Reply:
will this be the version we release for Serverless beta? I thought the log streaming was ready. @pivovarit